modules/ui/fields/input.js

```diff
@@ -413,7 +413,8 @@ export function uiFieldText(field, context) {
             var val = utilGetSetValue(input);
 
             if (field.type === 'tel') {
-                val = val.replace(/[^0-9+]/g, '');
+                val = val.replace(/^tel:\/\//, ''); // Remove 'tel://' prefix if present
+                val = val.replace(/^tel:/, ''); // Remove 'tel:' prefix if present
```

Comment:
Since these are regular expressions, you could combine these two steps by surrounding the slashes in `(\/\/)?`. (Technically, RFC&nbsp;3966 doesn’t allow slashes after the colon, but as https://github.com/openstreetmap/iD/issues/9559#issuecomment-1489139026 demonstrates, there’s probably enough confusion about that to go ahead and strip out those slashes too.)